test/suites/smoke-test-common/test-configuration-consistency.ts

```diff
@@ -69,10 +74,16 @@ describeSuite({
                     await api.query.authorityAssignment.collatorContainerChain(sessionIndex + 1)
                 ).toJSON();
 
-                // If we have container chain collators, is because we at least assigned min to orchestrator
-                if (
-                    Object.keys(authorities["orchestratorChain"]).length > config["minOrchestratorCollators"].toNumber()
-                ) {
+                const currentAuthorities = await api.query.session.validators();
+
+                const currentCollatorNumber = Math.min(currentAuthorities.length, config.maxCollators);
```

Comment:
In what case can the authorities be greater than max collators?

Reply:
probably in none, you are right

Reply:
maxCollators defines how many of currentAuthorities we take, where current authorities are invulnerables + staking collators. so we can have 100 potential candidatews, but maxCollators be only 70

Reply:
I see, thanks!